<?xml version="1.0"?>
<launch>
    <group ns="uav1">
        <node name="tf_broadcaster" pkg="borealis_tf_broadcaster" type="borealis_tf_broadcaster_node" output="screen">
            <!-- <remap from="system_map2local_odom_topic" to ="/gazebo/model_states"/> -->
            <!-- <remap from="system_map2body_topic" to ="ground_truth/state"/>  -->

            <remap from="local_odom_topic" to ="mavros/local_position/pose"/>
            <remap from="body2lidar_topic" to="/os_cloud_node/points"/>
            <remap from="body2camera_topic" to="t265/odom/sample"/>
            <remap from="systempose_topic" to="system_pose"/>

            <param name="broadcast_lidar" value="true"/>
            <param name="x_offset" value="2"/>
            <param name="y_offset" value="-2.5"/>
            <param name="z_offset" value="0"/>

        </node>
    </group>
</launch>

